src/bot_telegram.ts

```diff
@@ -37,9 +37,7 @@ bot.onText(/\/except (.+)/, async (msg, match) => {
 
   const chatMember = await bot.getChatMember(chatId, userId)
 
-  if (isAdminUser(chatMember)) {
-    // okay
-  } else {
+  if (!isAdminUser(chatMember)) {
```

Comment:
Also fixed this, because wtf @bartekpacia

Reply:
🤔
"that's in case I wanted to do something when the user is admin (in the future)"

Reply:
Yeah right

Reply:
You can always edit back if you need to

Reply:
sure master